### PR TITLE
Implement popup to inspect rendered features

### PR DESCRIPTION
**Pull request by aparlato**
Closes https://github.com/stamen/map-compare/issues/43

Implements a simple popup on click to inspect rendered features.

Popup closes on clicking the X or on clicking the map.

image

### CODE REVIEW

File: src/global.css [hunk 1]
```diff
@@ -67,3 +67,37 @@ button:not(:disabled):active {
 button:focus {
   border-color: #666;
 }
+
+.popup {
+  min-width: 180px;
+  padding-right: 12px;
+  max-height: 240px;
+  overflow: scroll;
```
Comment (ebrelsford) [inline src/global.css:75]: Could we change this to

```suggestion
  overflow: auto;
```

At least on my (ubuntu) machine I get scrollbars no matter what:

image

File: src/components/MapboxGlMap.svelte [hunk 2]
```diff
@@ -82,6 +84,44 @@
...
+
+    let html = '<div class="popup">';
+    for (const feature of dedupedFeatures) {
+      const { properties } = feature;
+      html += `<h2 class="popup-source-layer">${feature.source}: ${feature.sourceLayer}</h2>`;
+      if (properties && Object.keys(properties).length) {
+        Object.keys(properties).forEach(key => {
+          const propertyValue = properties[key];
+          html += `<p class="popup-property"><span class="popup-property-id">${key}:</span> <span class="popup-property-value">${propertyValue}</span></p>`;
+        });
```
Comment (ebrelsford) [inline src/components/MapboxGlMap.svelte:116]: What do you think about sorting by key here?

**Comment by aparlato** [comment]
Thanks! Took all your suggestions! Back to you for 👀  & ✅ !

**Review by ebrelsford — APPROVED** [review]
:tada:

**Pull request merged**